### ISSUE TITLE
Is it possible to patch Nuke::setSplashScreen instead of QSplashScreen::setPixmap

### ISSUE THREAD

**Issue by Ahuge** [heavyimage/OverrideQtSplashscreen#2]
That way we could scope it to the nuke splash screen and not all Qt ones.

Also we'd be able to do gif/movies because we just need to return a QSplashScreen.

I don't know if we are able to patch the Nuke namespace but I'd assume probably through some hackery?

**Comment by heavyimage** [comment]
Agreed it'd probably be better to just scope it to nuke for safety though It's kinda exciting to think this would work for all Qt apps (the pranking potential feels better, right?).  Sadly nuke doesn't have it's own splash screen function; as you can see in the readme from the results of looking at the (un mangled) symbol table it just directly calls the QT one.  There might be a way to only apply the substitution if we were *inside nuke* but I'm not quite sure how to detect that.

To do gifs is a bit [more advanced](https://forum.qt.io/topic/15658/solved-how-to-play-gif-animation-in-qgraphicsview-widget); worth a shot though!
